crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -3089,32 +3130,48 @@ impl<Codec: RawEntryCodec> StateMachine<Codec> {
         Ok(())
     }
 
-    async fn do_append_journal_entry<S: StateStorage>(
+    async fn append_journal_entry<S: StateStorage>(
         ctx: &mut StateMachineApplyContext<'_, S>,
         invocation_id: InvocationId,
         // We pass around the invocation_status here to avoid an additional read.
         // We could in theory get rid of this here (and in other places, such as StoreDeploymentId),
         // by using a merge operator in rocksdb.
-        previous_invocation_status: InvocationStatus,
+        mut previous_invocation_status: InvocationStatus,
         entry_index: EntryIndex,
         journal_entry: EnrichedRawEntry,
     ) -> Result<(), Error> {
         debug_if_leader!(
             ctx.is_leader,
             restate.journal.index = entry_index,
             restate.invocation.id = %invocation_id,
-            "Effect: Write journal entry {:?} to storage",
```

Comment:
In the `send_ingress_response` method we are still prefixing with "Effect:". Should we remove it there as well?